test/Altinn.Notifications.Email.IntegrationTests/Integrations/EmailSendingConsumerTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System.Text.Json;
+
+using Altinn.Notifications.Email.Core;
+using Altinn.Notifications.Email.Integrations.Configuration;
+using Altinn.Notifications.Email.Integrations.Consumers;
+using Altinn.Notifications.Email.IntegrationTests.Utils;
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+using Moq;
+
+using Xunit;
+
+namespace Altinn.Notifications.Email.IntegrationTests.Integrations;
+
+public class EmailSendingConsumerTests : IDisposable
+{
+    private const string TestTopic = "email-sending";
```

Comment:
a random guid ensures we never delete anything of value. This wouldn't match our naming properties, but who knows what people will use as topics in a dev context.

Reply:
Yeah, I can rewrite the test a bit to generate random topic name.